docs/commands/README.md

```diff
@@ -730,22 +727,21 @@ Upload workers to hosts, described in 'hosts' property in fluence.yaml
 
 ```
 USAGE
-  $ fluence workers upload [WORKER-NAMES] [--no-input] [--relay <value>] [--timeout <value>] [--ttl <value>] [-k
-    <value>] [--off-aqua-logs] [-k <value>]
+  $ fluence workers upload [WORKER-NAMES] [--no-input] [--relay <value>] [--ttl <value>] [--dial-timeout <value>]
+    [--particle-id] [-k <value>] [--off-aqua-logs] [--privKey <value>]
 
 ARGUMENTS
   WORKER-NAMES  Names of workers to deploy (by default all workers from 'hosts' property in fluence.yaml are deployed)
 
 FLAGS
   -k, --key-pair-name=<name>  Key pair name
-  -k, --privKey=<value>       !WARNING! for debug purposes only. Passing private keys through flags is unsecure
+  --dial-timeout=<value>      [default: 60000] Timeout for Fluence js-client
   --no-input                  Don't interactively ask for any input from the user
   --off-aqua-logs             Turns off logs from Console.print in aqua and from IPFS service
-  --relay=<multiaddr>         Relay node multiaddr
-  --timeout=<milliseconds>    [default: 60000] Timeout used for command execution
-  --ttl=<milliseconds>        Sets the default TTL for all particles originating from the peer with no TTL specified. If
-                              the originating particle's TTL is defined then that value will be used If the option is
-                              not set default TTL will be 60000
+  --particle-id               Print particle ids when running Fluence js-client
+  --privKey=<value>           !WARNING! for debug purposes only. Passing private keys through flags is unsecure
+  --relay=<value>             Relay for Fluence js-client to connect to
+  --ttl=<value>               [default: 60000] TTL for Fluence js-client
```

Comment:
```suggestion
  --ttl=<value>               [default: 60000] Particle Time To Live since 'now'. After that, particle is expired and not processed.
```